Sources/Classes/core/Models/InAppPurchaseResponse.swift

```diff
@@ -0,0 +1,73 @@
+import Foundation
+import StoreKit
+
+public enum MAProductResponseError: String, Codable, MiniAppErrorProtocol {
+
+    case productNotFound
+    case failedToConformToProtocol
+
+    var name: String {
+        return self.rawValue
+    }
+
+    public var description: String {
+        switch self {
+        case .productNotFound:
+        return "Host app Error"
+        case .failedToConformToProtocol:
+        return "Host app failed to implement required interface"
+        }
+    }
+}
+
+public enum MAProductResponseStatus: String, Codable {
+    case purchased = "PURCHASED"
+    case failed = "FAILED"
+    case restored = "RESTORED"
```

Comment:
I think the type in the JS SDK didn't have a field for the status? So how is this getting communicated to the Mini App?

Reply:
Yeah, updating JS SDK Side now. Thanks @corycaywood